.babelrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Let's leave out these changes to babel etc. in this PR. I'm going to upgrade the package to ESM and get rid of all this babel junk.